graph/db/sql_store.go

```diff
@@ -3539,7 +3557,11 @@ func buildNodeWithBatchData(dbNode sqlc.GraphNode,
 				"signed fields: %w", err)
 		}
 		if len(recs) != 0 {
-			node.ExtraOpaqueData = recs
+			if v == lnwire.GossipVersion1 {
+				node.ExtraOpaqueData = recs
```

Comment:
perhaps we could add a comment to `ExtraOpaqueData`'s docstring to mention it's only used for v1 nodes